tasty-query/shared/src/main/scala/tastyquery/Trees.scala

```diff
@@ -542,7 +536,8 @@ object Trees {
     *
     * { bindings; expr }
     */
-  final case class Inlined(expr: Tree, caller: Option[TypeIdent], bindings: List[Tree])(span: Span) extends Tree(span) {
+  final case class Inlined(expr: TermTree, caller: Option[TypeIdent], bindings: List[Tree])(span: Span)
```

Comment:
no idea if this would type check, but maybe it makes sense
```suggestion
  final case class Inlined(expr: TermTree, caller: Option[TypeIdent], bindings: List[DefTree])(span: Span)
```